content/docs/reference-react-component.md

```diff
@@ -122,13 +122,27 @@ You can also return multiple items from `render()` using an array:
 ```javascript
 render() {
   return [
-    <li key="A">First item</li>,
-    <li key="B">Second item</li>,
-    <li key="C">Third item</li>,
+    <li key="A">First item</li>,
+    <li key="B">Second item</li>,
+    <li key="C">Third item</li>,
   ];
 }
 ```
 
+The same can also be accomplished using [fragments](/docs/fragments.html):
```

Comment:
... which don't require keys for static items:

(and then let's remove keys)

Reply:
Should there be a note here "New in React 16.2"?

Reply:
The "New in React 16.2" I assumed was already covered in the linked document about details of this feature. Didn't want to attempt to cover too much detail here.

Reply:
We can say "Since React 16.2.0, ..."

"React 16.2.0" could be a link to the blog post